3_viz/src/1_Figure1_ModelComps.R

```diff
@@ -0,0 +1,61 @@
+#### This script plots up our evaluation data for Figure 1.
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
```

Comment:
I don't think you need stringr, sbtools, or whisker in this function/script, do you?